akamai/src/main/java/nl/nn/adapterframework/extensions/akamai/NetStorageSender.java

```diff
@@ -96,6 +96,11 @@
 
 	@Override
 	public void configure() throws ConfigurationException {
+		Parameter urlParameter = new Parameter();
+		urlParameter.setName(getUrlParam());
+		urlParameter.setSessionKey("urlParam");
+		addParameter(urlParameter);
```

Comment:
Waarom is dit een parameter? Om hem te kunnen overriden? Ik snap eigenlijk niet wat er gebeurt

Reply:
Het kan geen constante zijn, dan is het niet thread safe... vandaar dat het inputbericht altijd 'automatisch' naar de url parameter veranderd wordt.